src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -1,47 +1,170 @@
 package core.basesyntax;
 
 import java.util.List;
+import java.util.Objects;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
+    public MyLinkedList() {
+
+    }
```

Comment:
The constructor implementation begins with an empty line, which should be removed according to the checklist.